### PR TITLE
fix: add Vaishnav Shyam

### PR DESCRIPTION
**Pull request by vaishnav-sh**
## Changes proposed
Added new vaishnav-sh.json file in `public/data`

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/893"></a>

### CODE REVIEW

File: public/data/vaishnav-sh.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+    "name": "Vaishnav Shyam",
+    "type": "personal",
+    "bio": "Frontend developer | open source enthusiast",
```
Comment (syedsohan) [inline public/data/vaishnav-sh.json:4]:
```suggestion
    "bio": "Frontend developer | Open source enthusiast",
```

**Review by syedsohan — CHANGES_REQUESTED** [review]

**Review by syedsohan — CHANGES_REQUESTED** [review]
Change the PR title to - `fix: add Vaishnav Shyam`

File: public/data/vaishnav-sh.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "milestones": [
+      {
+        "title": "Graduated as a Bachelor in Computer Applications",
+        "date": "2021",
+        "icon": "Graduation Hat",
+        "color": "blue",
+        "description": "Graduated as a Bachelor in Computer Applications"
+      }
+    ]
+  }
```
Comment (krishguptadev) [inline public/data/vaishnav-sh.json:32]:
```suggestion
}

```

**Review by emmalearnscode — APPROVED** [review]

**Review by ssensalo — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**